config.json

```diff
@@ -40,7 +40,7 @@
       "description": "MsBuild logging options.",
       "valueType": "passThrough",
       "values": [],
-      "defaultValue": "/l:BinClashLogger,Tools/net46/Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log"
```

Comment:
Those failures indicate clashes.  That's just how the logger fails.  Investigate them.

Here's an example:
```
16:33:34 Error : Multiple projects built twice with the same target path D:\j\workspace\x64_release_w---0575cb46\bin\tests\Windows_NT.x64.Release\Common\Desktop.Coreclr.TestWrapper\Desktop.Coreclr.TestWrapper\Coreclr.TestWrapper.dll.
16:33:34 D:\j\workspace\x64_release_w---0575cb46\tests\runtest.proj -->
16:33:34   Global Properties:
16:33:34     __BuildType = Release
16:33:34     __BuildArch = x64
16:33:34     __BuildOS = Windows_NT
16:33:34     RestoreDefaultOptimizationDataPackage = false
16:33:34     UsePartialNGENOptimization = false
16:33:34     CLRTestPriorityToBuild = 1
16:33:34     TargetsWindows = true
16:33:34     BuildWrappers = true
16:33:34 D:\j\workspace\x64_release_w---0575cb46\tests\runtest.proj -->
16:33:34   Global Properties:
16:33:34     __BuildType = Release
16:33:34     __BuildArch = x64
16:33:34     __BuildOS = Windows_NT
16:33:34     RestoreDefaultOptimizationDataPackage = false
16:33:34     UsePartialNGENOptimization = false
16:33:34     CLRTestPriorityToBuild = 1
16:33:34     TargetsWindows = true
16:33:34     BuildWrappers = true
16:33:34     _CMDDIR = 
16:33:34 D:\j\workspace\x64_release_w---0575cb46\tests\runtest.proj -->
16:33:34   Global Properties:
16:33:34     __BuildType = Release
16:33:34     __BuildArch = x64
16:33:34     __BuildOS = Windows_NT
16:33:34     RestoreDefaultOptimizationDataPackage = false
16:33:34     UsePartialNGENOptimization = false
16:33:34     CLRTestPriorityToBuild = 1
16:33:34     TargetsWindows = true
16:33:34     BuildWrappers = true
16:33:34     _CMDDIR = D:\j\workspace\x64_release_w---0575cb46\bin\tests\Windows_NT.x64.Release\Regressions\expl_double
16:33:34 D:\j\workspace\x64_release_w---0575cb46\bin\tests\Windows_NT.x64.Release\TestWrappers\Regressions.expl_double\Regressions.expl_double.XUnitWrapper.csproj -->
16:33:34   Global Properties:
16:33:34     __BuildType = Release
16:33:34     __BuildArch = x64
16:33:34     __BuildOS = Windows_NT
16:33:34     RestoreDefaultOptimizationDataPackage = false
16:33:34     UsePartialNGENOptimization = false
16:33:34     CLRTestPriorityToBuild = 1
16:33:34     TargetsWindows = true
16:33:34     BuildWrappers = true
16:33:34     _CMDDIR = D:\j\workspace\x64_release_w---0575cb46\bin\tests\Windows_NT.x64.Release\Regressions\expl_double
16:33:34 D:\j\workspace\x64_release_w---0575cb46\tests\src\Common\Desktop.Coreclr.TestWrapper\Desktop.Coreclr.TestWrapper.csproj
16:33:34   Global Properties:
16:33:34     __BuildType = Release
16:33:34     __BuildArch = x64
16:33:34     __BuildOS = Windows_NT
16:33:34     RestoreDefaultOptimizationDataPackage = false
16:33:34     UsePartialNGENOptimization = false
16:33:34     CLRTestPriorityToBuild = 1
16:33:34     TargetsWindows = true
16:33:34     BuildWrappers = true
16:33:34     _CMDDIR = D:\j\workspace\x64_release_w---0575cb46\bin\tests\Windows_NT.x64.Release\Regressions\expl_double
16:33:34     OSGroup = Windows_NT
16:33:34 
16:33:34 D:\j\workspace\x64_release_w---0575cb46\tests\runtest.proj -->
16:33:34   Global Properties:
16:33:34     __BuildType = Release
16:33:34     __BuildArch = x64
16:33:34     __BuildOS = Windows_NT
16:33:34     RestoreDefaultOptimizationDataPackage = false
16:33:34     UsePartialNGENOptimization = false
16:33:34     CLRTestPriorityToBuild = 1
16:33:34     TargetsWindows = true
16:33:34     BuildWrappers = true
16:33:34 D:\j\workspace\x64_release_w---0575cb46\tests\runtest.proj -->
16:33:34   Global Properties:
16:33:34     __BuildType = Release
16:33:34     __BuildArch = x64
16:33:34     __BuildOS = Windows_NT
16:33:34     RestoreDefaultOptimizationDataPackage = false
16:33:34     UsePartialNGENOptimization = false
16:33:34     CLRTestPriorityToBuild = 1
16:33:34     TargetsWindows = true
16:33:34     BuildWrappers = true
16:33:34     _CMDDIR = 
16:33:34 D:\j\workspace\x64_release_w---0575cb46\tests\runtest.proj -->
16:33:34   Global Properties:
16:33:34     __BuildType = Release
16:33:34     __BuildArch = x64
16:33:34     __BuildOS = Windows_NT
16:33:34     RestoreDefaultOptimizationDataPackage = false
16:33:34     UsePartialNGENOptimization = false
16:33:34     CLRTestPriorityToBuild = 1
16:33:34     TargetsWindows = true
16:33:34     BuildWrappers = true
16:33:34     _CMDDIR = D:\j\workspace\x64_release_w---0575cb46\bin\tests\Windows_NT.x64.Release\baseservices\compilerservices
16:33:34 D:\j\workspace\x64_release_w---0575cb46\bin\tests\Windows_NT.x64.Release\TestWrappers\baseservices.compilerservices\baseservices.compilerservices.XUnitWrapper.csproj -->
16:33:34   Global Properties:
16:33:34     __BuildType = Release
16:33:34     __BuildArch = x64
16:33:34     __BuildOS = Windows_NT
16:33:34     RestoreDefaultOptimizationDataPackage = false
16:33:34     UsePartialNGENOptimization = false
16:33:34     CLRTestPriorityToBuild = 1
16:33:34     TargetsWindows = true
16:33:34     BuildWrappers = true
16:33:34     _CMDDIR = D:\j\workspace\x64_release_w---0575cb46\bin\tests\Windows_NT.x64.Release\baseservices\compilerservices
16:33:34 D:\j\workspace\x64_release_w---0575cb46\tests\src\Common\Desktop.Coreclr.TestWrapper\Desktop.Coreclr.TestWrapper.csproj
16:33:34   Global Properties:
16:33:34     __BuildType = Release
16:33:34     __BuildArch = x64
16:33:34     __BuildOS = Windows_NT
16:33:34     RestoreDefaultOptimizationDataPackage = false
16:33:34     UsePartialNGENOptimization = false
16:33:34     CLRTestPriorityToBuild = 1
16:33:34     TargetsWindows = true
16:33:34     BuildWrappers = true
16:33:34     _CMDDIR = D:\j\workspace\x64_release_w---0575cb46\bin\tests\Windows_NT.x64.Release\baseservices\compilerservices
16:33:34     OSGroup = Windows_NT
```

In this case the logger identified that Coreclr.TestWrapper.dll was built at least twice.  The reason it was built twice is because the values of the global property _CMDDIR were different in each build.  If those builds could happen at the same time then they'd race and produce failures.  Just glancing through the log it seems a lot of these are the same flavor of error, so I suspect some project/targets is leaking the value of _CMDDIR down to places it shouldn't.  Don't use global properties on project references unless they'll effect the output directory otherwise you'll introduce race conditions to the build.